pkg/sfu/forwarder_test.go

```diff
@@ -1007,49 +1007,49 @@ func TestForwarderAllocateNextHigher(t *testing.T) {
 	// ask again, should return not boosted as there is no room to go higher
 	f.currentLayers.Spatial = 2
 	f.currentLayers.Temporal = 1
-	result, boosted = f.AllocateNextHigher(ChannelCapacityInfinity, nil, bitrates, false)
+	result, boosted = f.AllocateNextHigher(100_000_000, nil, bitrates, false)
 	require.Equal(t, expectedResult, result)
 	require.Equal(t, expectedResult, f.lastAllocation)
 	require.Equal(t, expectedTargetLayers, f.TargetLayers())
 	require.False(t, boosted)
 
 	// turn off everything, allocating next layer should result in streaming lowest layers
 	disable(f)
-	f.lastAllocation.isDeficient = true
-	f.lastAllocation.bandwidthRequested = 0
+	f.lastAllocation.IsDeficient = true
+	f.lastAllocation.BandwidthRequested = 0
 
-	expectedTargetLayers = VideoLayers{
+	expectedTargetLayers = buffer.VideoLayer{
 		Spatial:  0,
 		Temporal: 0,
 	}
 	expectedResult = VideoAllocation{
-		isDeficient:         true,
-		bandwidthRequested:  2,
-		bandwidthDelta:      2,
-		bandwidthNeeded:     bitrates[2][1],
-		bitrates:            bitrates,
-		targetLayers:        expectedTargetLayers,
-		requestLayerSpatial: expectedTargetLayers.Spatial,
-		maxLayers:           DefaultMaxLayers,
-		distanceToDesired:   2.25,
-	}
-	result, boosted = f.AllocateNextHigher(ChannelCapacityInfinity, nil, bitrates, false)
```

Comment:
This is the constant from stream allocator which cannot be used in these tests from 'streamallocator' name space. Just using the value for now to get around it.

Reply:
sometimes circular dependencies in tests can be fixed using `_test` packages but it doesn't look like it would work here because the test accesses private values